ui/packages/docs/docs/guides/building_a_simulation_basic_tutorial.md

```diff
@@ -644,6 +644,94 @@ With this, the batterying will only occur if Xiangling has less than 70 energy a
 
 ![after batterying](sim_17.png)
 
+## Rotation optimization
+
+In this section we're going to go over some common rotation optimization tricks.
+
+### Maximizing number of Raiden attacks in burst
+
+I mentioned earlier that I put `raiden attack:15;` because I didn't know exactly how many normal attacks Raiden can squeeze in just spamming left mouse click (no fancy combos here) during her burst duration. Now we're going to look at how we can use debug to optimize that.
+
+:::info
+Although a lot of effort has gone into making sure gcsim has accurate frame counts, it's always a good idea to reference vs a video of your own gameplay to see what actions are possible. Keeping in mind that gcsim frames are as optimal as possible (i.e. perfect frame cancels) which could be significantly faster than what you may be able to execute
+:::
+
+So to start, we're going to generate a sample using `Min Seed` again. This time make sure that the options are set to show Advanced as follows (we're going to need some of the options here, namely `status`):
+
+![advanced options](opt_1.png)
+
+Next scroll down to the first instance where Raiden uses burst (roughly 6.7s in). You'll see a status line saying: `raidenburst status mod added`. We're going to click on the tiny little icon to the left of it, this is going to highlight the F column like so:
+
+![show buff duration](opt_2.png)
+
+This highlight on the F column is a little helper that shows how long that the particular buff that you click on lasts. When the blue highlight fades is when that buff expires. So now we can scroll down and find where the Raiden burst buff expires:
+
+![raiden burst buff expires](opt_3.png)
+
+From this picture, we can see that the last normal attack started on frame 976, and the buff expires 980. This shows that our 15 normal attacks lines up perfectly and fits into the burst duration perfectly! We didn't need to do anything.
+
+### Optimizing particles
+
+Another important aspect of optimizing a rotation involves taking a look at energy and making sure the right character is catching the particle. If you are familiar with this particular team, you may know that Xiangling is one of the more energy hungry character of the team. One potential optimization we can make is look at if there are ways to increase her energy gain.
```

Comment:
probably meant: particle -> particles
hungry character -> hungry characters
is look at if there are ways -> is to look for ways